datastorage.lua

```diff
@@ -58,8 +61,8 @@ end
 local function initDataDir()
     local sub_data_dirs = {
         "cache", "clipboard",
-        "data", "data/dict", "data/tessdata",
-        "history", "ota", "plugins",
+        "data", "data/dict", "data/tessdata", "docsettings",
+        "history", "ota", "patches", "plugins",
```

Comment:
Any special reason to have added "patches" in there ?
I think it was supposed to be explicitely created by the hardcore users who would need user patches, and the absence of this directory keeps the PatchManager plugin disabled, so to not lead softcore users into temptation :)
(Pinging @zwim)